app/helpers/diaper_drive_helper.rb

```diff
@@ -0,0 +1,7 @@
+module DiaperDriveHelper
+  def is_virtual(diaper_drive)
+    return 'No' if diaper_drive.blank?
```

Comment:
I think raising an error would be preferrable in this case, returning 'No' does not inform us that we used the method wrong by passing it `nil`